process/graduation_checklist.md

```diff
@@ -0,0 +1,55 @@
+# ASWF Project Graduation Template
+
+Text below in _italics_ are guidelines for answering the various
+questions.
+
+- **Project Name**:
+
+- **Criteria**
+
+  - **Demonstrate having a healthy number of committers from a diverse
+    contributor base.**
+
+    _Showcase the total number of committers as well as the diversity
+    of committers from an organizational standpoint. Link to Pull
+    Request reviews/approvals, or other indications of committer-level
+    activities. Each ASWF project is on [LF Dev Analytics] which is a 
+    good resource to use for this._
+ 
+  - **Demonstrate a substantial ongoing flow of commits and merged
+    contributions, authored by a healthy number of diverse
+    contributors.**
+  
+    _Showcase the total number of contributors since the project
+    entered incubation, as well as the diversity of contributors from
+    an organizational standpoint. Pointing to GitHub or Dev Analytics
+    graphs are good ideas ( i.e.
+    https://github.com/imageworks/OpenColorIO/graphs/contributors ).
+    Feel free to add other context if relevant. 
+    Each ASWF project is on [LF Dev Analytics] which is a 
+    good resource to use for this._
+
+  - **Document current project owners and current and emeritus
+    committers in a COMMITTER file or similarly visible system. A copy
+    of the project’s charter (or other authorized governance document)
+    will be included or linked to in visible location.**
+  
+    _Link to GOVERNAME.md, COMMITTERS.md, and TSC Charter_
```

Comment:
What's `GOVERNAME.md`? I'm not familiar with that. I think `CODEOWNERS` is a good example to list here, as Github [uses that file for automated checks](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners).

Reply:
First off - there was a typo ( should be ```GOVERNANCE.md``` )

So generally this doc is used to outline roles in the community, how decisions are made, and any other policies of the project - in other word how governance works in the project. It's usually fairly simple - OpenEXR is a good example here...

https://github.com/AcademySoftwareFoundation/openexr/blob/master/GOVERNANCE.md

Let me know if this makes sense.

Reply:
Ah right, ok.

Still might be good to mention `CODEOWNERS`? Or is there a reason to have both `CODEOWNERS` (for Github integration) as well as `COMMITTERS.md`?